docs/.vuepress/helpers.js

```diff
@@ -57,10 +59,23 @@ function getPrettyFrameworkName(framework) {
  */
 function getThisDocsVersion() {
   if (docsVersion === null) {
+
     const branchName = execa.sync('git rev-parse --abbrev-ref HEAD', { shell: true }).stdout;
 
-    if (versionFromBranchRegExp.test(branchName)) {
+    if (versionFromBranchRegExp.test(branchName) === true) {
       docsVersion = branchName.match(versionFromBranchRegExp)[1];
+    } else if (branchDev.test(branchName) || branchProdDocsLatestRegexp.test(branchName) === true) {
+
+      const allRemoteBranches = execa.sync('git ls-remote --heads origin ', { shell: true }).stdout;
+      const versionsFound = allRemoteBranches.split('\n')
+        .filter(item => item.includes('prod-docs'))
+        // avoids the case when the branch is named `prod-docs/latest`
+        .filter(item => item.match(/\d+\.\d+/))
+        .map(item => item.match(/\d+\.\d+/)[0]);
+
+      const maximumVersionFound = Math.max(...versionsFound);
+
+      docsVersion = maximumVersionFound.toString();
```

Comment:
Also, I'd consider utilizing the env (`LATEST_VERSION`) variable that you already created in [GH workflow](https://github.com/handsontable/handsontable/pull/11174/files#diff-24418a9942057d986cc1cd3ac174aecc5a21541647749be9e89be8735a12a89dR36).
```suggestion
    } else if (DOCS_LATEST_VERSION) {
      return process.env.DOCS_LATEST_VERSION;
```
Also, I'd add a `DOCS_` prefix if we want to go there.

Reply:
Good point. I changed everywhere, including workflow, router.ts and Netlify site environment variable

Reply:
One more thing, except of renaming - I need to investigate this, beacuse the last time I could not use process.env inside vuepress

Reply:
Fixed and tested, DOCS_LATEST_VERSION is set from the pipeline environment